frontend/components/JoinNextPatch/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const JoinNextPatch = ({ data }) => {
+  const { title, description } = data;
+  return (
+    <div className="join-next-patch border-4 border-dashed border-c100 w-3/4 h-1/2 m-auto">
```

Comment:
dont't use any width or height at all ... let the component take the width and height of it's container